releasenotes/notes/asgi-resource-obfuscation-404-d77e8abd660c8132.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    asgi: Obfuscate resource names on 404 spans when ``DD_ASGI_OBFUSCATE_404_RESOURCE`` is enabled (disabled by default).
```

Comment:
I think the asgi integrations doc should also be updated to showcase this new config option: https://github.com/DataDog/dd-trace-py/blob/79677cc65aaece267864c324dcb01e8c5533904e/ddtrace/contrib/asgi.py#L43